aunt-leahs-app/src/pages/VolunteerHomePage.jsx

```diff
@@ -29,7 +29,13 @@ const VolunteerHomePage = ({ setVolunteerLocation, selectedLocation, setCurrentP
     }
 
     const getLocations = async () => {
-		const response = await fetch('http://localhost:7071/api/location');
+		const response = await fetch(locationsEndpoint, {
+			method: 'GET',
+			headers: {
+				'Content-Type':'application/json',
```

Comment:
Why does this GET request require these headers while others don't?

Reply:
I think the others weren't in there when I originally branched off. They should all need them if we configure the CORS for the volunteer functions too

Reply:
This is a valid concern, but can we merge and have this as a separate issue?